src/UI/MainMenuOptions.cpp

```diff
@@ -69,30 +69,44 @@ void MainMenuOptions::init()
     cmbResolution.size(200, 0);
     cmbResolution.maxDisplayItems(1);
 
-    cmbResolution.clearSelection();
-    {
-        extern SDL_Window* underlyingWindow;
-        const auto display_index = SDL_GetWindowDisplayIndex(underlyingWindow);
-        const auto num_resolutions = SDL_GetNumDisplayModes(display_index);
-        std::ostringstream ss;
-        std::string resolutionText{};
-        SDL_DisplayMode active_mode{};
-        for (int i = 0; i < num_resolutions; ++i)
-        {
-            SDL_DisplayMode cur_mode{};
-            SDL_GetDisplayMode(display_index, i, &cur_mode);
-            SDL_GetCurrentDisplayMode(display_index, &active_mode);
-            ss << cur_mode.w << 'x' << cur_mode.h << 'x' << cur_mode.refresh_rate;
-            if (cur_mode.w == active_mode.w && cur_mode.h == active_mode.h && cur_mode.refresh_rate == active_mode.refresh_rate)
-            {
-                currentResolutionSelection = i;
-                resolutionText = ss.str();
-            }
-            cmbResolution.addItem(ss.str(), i);
-            ss.str("");
-        }
-    }
-    cmbResolution.selectionChanged().connect(this, &MainMenuOptions::onVideoOptionsChanged);
+	cmbResolution.clearSelection();
+	{
+		const auto default_resolution = std::to_string(constants::MINIMUM_WINDOW_WIDTH) + 'x' + std::to_string(constants::MINIMUM_WINDOW_HEIGHT) + 'x' + std::to_string(60);
+		cmbResolution.text(default_resolution);
+	}
```

Comment:
If this is removed later, the removal could have been squashed with the addition.

Reply:
I don't understand. This was removed in a later commit. Using the `Start Review` button instead of commit-by-commit only shows you the most-up-to-date versions of each file, thus, you would have never seen it's addition to comment on, just the deletion.

Also, I'm pretty sure it's addition was part of that big commit that I couldn't separate.

Reply:
I added the comment after viewing the removal.

My point was more that it could have been cleaned up before review.

Reply:
Ah